test/integration/targets/cloudscale_server/tasks/tests.yml

```diff
@@ -42,6 +45,7 @@
       - server is changed
       - server.state == 'running'
       - server.server_groups.0.name == '{{ cloudscale_resource_prefix }}-group-1'
+      - server.zone.slug == '{{ cloudscale_test_zone }}'
```

Comment:
not a blockerl, just fyi: the templating is superfluous --> `server.zone.slug == cloudscale_test_zone`

Reply:
Ah, that makes sense, thanks. I'll keep that in mind for future changes.